### ISSUE TITLE
Data is duplicated when reading and writing a GLB

### ISSUE THREAD

**Issue by javagl** [javagl/JglTF#106]
I just noticed that an operation as trivial as
```java
GltfModelReader r = new GltfModelReader();
GltfModel m = r.read(Paths.get("input.glb"));

GltfModelWriter w = new GltfModelWriter();
w.writeBinary(m, new File("output.glb"));
```
causes buffer data to be duplicated (and the model becoming twice as large as before). 

While preparing the model to be written as binary, it does not take into account that the images already refer to parts of the buffer in the input.

This has to be fixed ASAP.

**Comment by javagl** [comment]
This should be fixed with https://github.com/javagl/JglTF/pull/108 , but I'll leave it open for now, probably until a new version is released...

**Comment by Jeongyong-park** [comment]
> This should be fixed with #108 , but I'll leave it open for now, probably until a new version is released...

When is the day of next version released?

**Comment by javagl** [comment]
Well... I don't plan/schedule regular releases. To some extent, it depends on how much of my spare time I (can) spend here. 

But there indeed have been quite changes since the last release. (To keep track of that, I actually started a change log recently, and the list for the current 'SNAPSHOT' version at https://github.com/javagl/JglTF/blob/master/CHANGES.md#204-snapshot is already pretty long...).

I'll try to create a new release next week (no promises, but ... should be doable... 🤞 )

**Comment by Jeongyong-park** [comment]
> Well... I don't plan/schedule regular releases. To some extent, it depends on how much of my spare time I (can) spend here.
> 
> But there indeed have been quite changes since the last release. (To keep track of that, I actually started a change log recently, and the list for the current 'SNAPSHOT' version at https://github.com/javagl/JglTF/blob/master/CHANGES.md#204-snapshot is already pretty long...).
> 
> I'll try to create a new release next week (no promises, but ... should be doable... 🤞 )

I'm still waiting for you are release next version. 😢😢😢

**Comment by javagl** [comment]
Yeah, there have been some delays (the work that I'm paid for, you know...). But I just released version 2.0.4, and it should be available in Maven Central soon.

**Comment by javagl** [comment]
So with version 2.0.4, the data should no longer be duplicated. This was fixed as part of a larger refactoring in https://github.com/javagl/JglTF/pull/108 , so let's hope that I didn't introduce any new bugs there, but for now, I think that this issue can be closed.
